app/src/main/java/com/example/vlad/financemanager/data/models/Category.java

```diff
@@ -2,38 +2,43 @@
 
 import java.io.Serializable;
 
-public class Category implements Serializable{
+public class Category implements Serializable {
     private int id;
     private String name;
     private int icon;
     private boolean isCustom;
     private boolean isInputCategory;
 
-    public Category(int id, String name, int icon, boolean isCustom, boolean isInputCategory){
+    public Category(int id, String name, int icon, boolean isCustom, boolean isInputCategory) {
         this.id = id;
         this.name = name;
         this.icon = icon;
         this.isCustom = isCustom;
         this.isInputCategory = isInputCategory;
     }
 
-    public Category(){
+    public Category() {
         this(0, "Unknown", 0, true, false);
     }
 
-    public boolean getIsCustom(){return  isCustom;}
-    public boolean getIsInputCategory(){return isInputCategory;}
+    public boolean getIsCustom() {
+        return isCustom;
+    }
+
+    public boolean getIsInputCategory() {
+        return isInputCategory;
+    }
 
-    public int getId(){
+    public int getId() {
         return id;
     }
 
     public String getName() {
         return name;
     }
 
-    public void setName(String name) throws IllegalArgumentException{
-        if(name != "")
+    public void setName(String name) throws IllegalArgumentException {
```

Comment:
`throws IllegalArgumentException ` - Dont do in a such way, especially for setter